lightspeed/modules/proc_create-lightspeed-instance.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: PROCEDURE
+
+[id="create-lightspeed-instance_{context}"]
+
+= Creating and deploying a {LightspeedShortName} instance
+
+Use this procedure to create and deploy a {LightspeedShortName} instance to your namespace.  
+
+.Prerequisites
+* You have created connection secrets for both {PlatformName} and {ibmwatsonxcodeassistant}.
+
+.Procedure
+
+. Go to {OCP}.
+. Select *Operators → Installed Operators*.
+. From the *Projects* list, select the namespace where you want to install the {LightspeedShortName} instance and where you created the connection secrets.
+. Locate and select the *{PlatformNameShort} (provided by Red Hat)* operator that you installed earlier.
```

Comment:
Why not say "Locate and select the Red Hat Ansible Automation Platform operator" instead of including the parenthetical "provided by Red Hat"? You could use an attribute too-- Red Hat {OperatorPlatform}

Reply:
Having read further, it seems that (provided by Red Hat) is actually in the UI? If so, please disregard.

Reply:
Yes, the 'provided by RH' is the UI label for the Lightspeed operator. Not making any change here.